components/match2/wikis/clashroyale/match_summary.lua

```diff
@@ -110,6 +110,7 @@ function CustomMatchSummary._createGame(game, gameIndex, date)
 		participantData.cards = participantData.cards or {}
 		local cards = Array.map(Array.range(1, NUM_CARDS_PER_PLAYER), function(idx)
```

Comment:
```suggestion
		local cards = Table.mapValues(Array.range(1, NUM_CARDS_PER_PLAYER), function(idx)
```

instead of the cast?

Reply:
even with the `Table.mapValues`/`Table.map` it still warns with
```Fields cannot be injected into the reference of `{ [integer]: string }` for `tower`. To allow injection, add `[any]: any` to the definition.```

Reply:
used type instead of cast now